### DISCUSSION TITLE
домен по сле установки просто на IP адрес

### DISCUSSION THREAD

**Discussion by Romanufa999** [totumonline/totum-ru-issues-and-discussions#610] [Как это сделать?] [answered]
добрый день.
В инструкции сказано
Вы сможете подключить домен позже, сделав A-запись и повторно запустив установщик ¡из той же папки из которой он был запущен при первоначальной установке!

У меня не сотрется то что я сейчас настроил ? данные останутся ??

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Если речь идет о V5 то данные останутся — будет получен сертификат на домен и ip изменен на указанный домен.

Установщик не может изменить один домен на другой — это делается по отдельной инструкции.
